sdk/daml-lf/validation/src/test/scala/com/digitalasset/daml/lf/validation/upgrade/UpgradesSpecBase.scala

```diff
@@ -1049,6 +324,57 @@ abstract class UpgradesSpec(val suffix: String)
     testPackagePair(v1Upload, v2Upload, uploadAssertion)
   }
 
+  def testPackagePairUpgradeCheck(
```

Comment:
I think you can delete that function and the one below now.

Reply:
done